code/ui/components/dropdown/DropdownOption.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+namespace TTTReborn.UI
+{
+    public partial class DropdownOption : TTTPanel
+    {
+        public Dropdown Dropdown { get; set; }
+
+        public Label TextLabel { get; private set; }
+
+        public Action<TTTPanel> OnSelect { get; set; }
+
+        public DropdownOption(Panel parent = null, string text = null) : base(parent)
+        {
+            Parent = parent ?? Parent;
+
+            TextLabel = Add.Label(text ?? "Select", "optiontext");
+        }
+
+        protected override void OnClick(MousePanelEvent e)
+        {
+            OnSelect?.Invoke(this);
+            Dropdown?.OnSelectOption(this);
+        }
+    }
+}
```

Comment:
Imo this can be in the same file as `Dropdown.cs`

Reply:
Wouldn't put it together, as we have a folder anyways.

Reply:
no worries, just a suggestion.